src/Elasticsearch.Net/Purify/Purify.cs

```diff
@@ -158,31 +163,34 @@ static PurifierMono()
 					BindingFlags.NonPublic | BindingFlags.Instance);
 			}
 
-			public Uri Purify(Uri uri)
+			Uri IPurifier.Purify(Uri uri)
 			{
-				var source = (string)mono_sourceField.GetValue(uri);
+				var source = (string) mono_sourceField.GetValue(uri);

```

Comment:
space after `(cast)` triggers my OCD, in most places there is no space after a cast in our codebase.